types/types.proto

```diff
@@ -356,6 +356,21 @@ message PodRemoveResponse {
   string cause  = 2;
 }
 
+message ContainerLogsRequest {
+  string  container =1;
+  bool    follow    =2;
+  bool    timestamps=3;
+  string  tail      =4;
+  string  since     =5;  
+  bool    stdout    =6;
+  bool    stderr    =7;
+
+}
+
+message ContainerLogsResponse {
+  string  log      =1;
```

Comment:
Yes,I've tried it with "repeated byte",but there seems no tyep as "byte".And "bytes" is OK